README.md

```diff
@@ -1,231 +1,38 @@
 <img src="figs/logo_01.png" width="150"/>
 
-
-**_v0.7.0-beta_**
-
 [![Build Status](https://travis-ci.org/HazyResearch/snorkel.svg?branch=master)](https://travis-ci.org/HazyResearch/snorkel)
 [![Documentation](https://readthedocs.org/projects/snorkel/badge/)](http://snorkel.readthedocs.io/en/master/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Getting Started
```

Comment:
Note: Before anything goes live (including as pre-release), we need to have something here- should just copy the content from the new website

Reply:
Idk if we want to maintain/sync content in two places. Maybe just a brief bit from the intro with a strong pointer to the webpage?

Reply:
Well I didn't say all the content, but the main points should be here, in case people bypass the main page. This can be done as a separate PR, but needs to be up!